luxonis_train/lightning/luxonis_lightning.py

```
@@ -878,6 +878,36 @@ def _evaluation_epoch_end(self, mode: Literal["test", "val"]) -> None:
                             sync_dist=True,
                         )
 
+                if self.trainer.is_global_zero:
+                    try:
+                        artifacts = metric.get_artifacts(computed)
+                    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)